client/client.go

```diff
@@ -227,11 +227,11 @@ func (c *Client) loginTotp(user, password, totpSeed string) error {
 	r = r.SetHeaderVerbatim("RequestVerificationToken", token).SetDebug(c.debug)
 	c.client = r
 	slog.Info("Logged in with TOTP", slog.String("user", user))
-	jwt, _, err := jwt.NewParser().ParseUnverified(c.currentToken, jwt.MapClaims{})
+	javaWebToken, _, err := jwt.NewParser().ParseUnverified(c.currentToken, jwt.MapClaims{})
```

Comment:
Wondering why that did not cause any issues until now ...

Reply:
This was only a linter complaint, the code compiles fine. Looks like golang's name resolution prefers local variables over package names…

It's a potential source of confusion, so renaming the variable seems reasonable.